src/App/Pages/Send/SendAddEditPage.xaml.cs

```diff
@@ -0,0 +1,229 @@
+using System;
+using System.Collections.Generic;
+using Bit.App.Resources;
+using Bit.Core.Abstractions;
+using Bit.Core.Enums;
+using Bit.Core.Utilities;
+using Xamarin.Forms;
+using Xamarin.Forms.PlatformConfiguration;
+using Xamarin.Forms.PlatformConfiguration.iOSSpecific;
+using Entry = Xamarin.Forms.Entry;
+
+namespace Bit.App.Pages
+{
+    public partial class SendAddEditPage : BaseContentPage
+    {
+        private readonly IBroadcasterService _broadcasterService;
+        
+        private SendAddEditPageViewModel _vm;
+        
+        public SendAddEditPage(
+            string sendId = null, 
+            SendType? type = null)
+        {
+            _broadcasterService = ServiceContainer.Resolve<IBroadcasterService>("broadcasterService");
+            InitializeComponent();
+            _vm = BindingContext as SendAddEditPageViewModel;
+            _vm.Page = this;
+            _vm.SendId = sendId;
+            _vm.Type = type;
+            _vm.Init();
+            if (Device.RuntimePlatform == Device.Android)
+            {
+                if (_vm.EditMode)
+                {
+                    ToolbarItems.Add(_removePassword);
+                    ToolbarItems.Add(_copyLink);
+                    ToolbarItems.Add(_shareLink);
+                    ToolbarItems.Add(_deleteItem);
+                }
+                _vm.EditorMargins = new Thickness(0,5,0,0);
+            }
+            if (Device.RuntimePlatform == Device.iOS)
```

Comment:
Seems lame, but you can save a CPU cycle on this eval if you prepend the `if` with an `else`

Reply:
Cycle saved!